<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntroWindow</class>
 <widget class="QMainWindow" name="IntroWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>키오스크</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="eatHereButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>350</width>
      <height>350</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>매장에서 식사</string>
    </property>
   </widget>
   <widget class="QPushButton" name="takeOutButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>310</y>
      <width>350</width>
      <height>350</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>테이크 아웃</string>
    </property>
   </widget>
   <widget class="QPushButton" name="goHomeButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>900</y>
      <width>700</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string>처음으로</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>681</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string>식사 방법을 선택해 주세요.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
